### ISSUE TITLE
dev-util/cargo-0.31.1-r666 fails to download crates

### ISSUE THREAD

**Issue by okias** [gentoo/gentoo-rust#382]
```
>>> Emerging (1 of 2) dev-util/cargo-0.31.1-r666::rust
 * Fetching files in the background.
 * To view fetch progress, run in another terminal:
 * tail -f /var/log/emerge-fetch.log
 * cargo-0.31.1.tar.gz SHA256 SHA512 WHIRLPOOL size ;-) ...                                [ ok ]
 * cargo-0.30.0-x86_64-unknown-linux-gnu.tar.xz SHA256 SHA512 WHIRLPOOL size ;-) ...       [ ok ]
>>> Unpacking source...
>>> Unpacking cargo-0.31.1.tar.gz to /var/tmp/portage/dev-util/cargo-0.31.1-r666/work
>>> Unpacking cargo-0.30.0-x86_64-unknown-linux-gnu.tar.xz to /var/tmp/portage/dev-util/cargo-0.31.1-r666/work
 * Crates will be fetched from crates.io.
>>> Source unpacked in /var/tmp/portage/dev-util/cargo-0.31.1-r666/work
>>> Preparing source in /var/tmp/portage/dev-util/cargo-0.31.1-r666/work/cargo-0.31.1 ...
>>> Source prepared.
>>> Configuring source in /var/tmp/portage/dev-util/cargo-0.31.1-r666/work/cargo-0.31.1 ...
>>> Source configured.
>>> Compiling source in /var/tmp/portage/dev-util/cargo-0.31.1-r666/work/cargo-0.31.1 ...
    Updating registry `https://github.com/rust-lang/crates.io-index`
warning: spurious network error (2 tries remaining): curl error: Could not resolve host: github.com

```

**Comment by gentoo90** [comment]
This ebuild needs `RESTRICT="network-sandbox"` in it. Just like in #340
